packages/toolkit/src/lib/use-instill-form/pick/pickComponentOutputFieldsFromInstillFormTree.tsx

```diff
@@ -164,15 +174,17 @@ export function pickComponentOutputFieldsFromInstillFormTree(
 
     return propertyValue && Array.isArray(arrayArrayData) ? (
       <div key={tree.path || tree.fieldKey} className="flex flex-col gap-y-2">
-        {arrayArrayData.map((data) => {
-          return pickComponentOutputFieldsFromInstillFormTree({
-            ...props,
-            tree: tree.items,
-            data: {
-              [tree.fieldKey as string]: data,
-            },
-          });
-        })}
+        {arrayArrayData.map((data, idx) => (
+          <React.Fragment key={`${tree.path || tree.fieldKey}-${idx}`}>
```

Comment:
I'm not sure you need to use these wrappers here. The returned value from the `pickComponentOutputFieldsFromInstillFormTree` already has the `key` prop.

Reply:
addressed.